mlflow/genai/scorers/builtin_scorers.py

```diff
@@ -19,6 +19,26 @@
 GENAI_CONFIG_NAME = "databricks-agent"
 
 
+def _validate_databricks_environment(scorer_name: str) -> None:
+    """Validate that we're in a Databricks environment.
```

Comment:
```suggestion
    """Validate that the current environment is Databricks.
```

Can we replace `we're`?